src/components/common/footer/Footer.tsx

```diff
@@ -1,42 +1,44 @@
 import React, { FC } from "react";
 import {
-	FooterContainer,
-	FooterBar,
-	LogoContainer,
-	KnowUs,
-	Email,
-	NavLinksContainer,
-	NavLink,
-	RightsReserved,
+  FooterContainer,
+  FooterBar,
+  LogoContainer,
+  KnowUs,
+  Email,
+  NavLinksContainer,
+  NavLink,
+  RightsReserved,
 } from "./Footer.css";
 import SocialIcons from "../social-icons/SocialIcons";
 import icon from "../../../assets/footer/QG-Logo-V3-Gold-Transparent-PNG-1.png";
 
+// import Contact from "../../contact/Contact";
```

Comment:
Can we remove this if it's not being used?

Reply:
+1 on this one. Will remember to not do this, myself